src/domain/Loan.ts

```diff
@@ -72,7 +75,14 @@ export class Installment {
     this.createdAt = createdAt;
     this.upDatedAt = upDatedAt;
     this.status = status;
+    this.paidAt = paidAt;
   }
+
+  paidInstallment = (): void => {
```

Comment:
@thaismichelini sugestão: acho que faz mais sentido o nome da function ficar **payInstallment**
```suggestion
  payInstallment = (): void => {
```

O que achas?